security-considerations.md

```diff
@@ -2,6 +2,9 @@
 
 ## Price considerations
 
+The following considerations make some claims that are important for the security of the code.
+References in square brackets can be found in the code as `\\[*]`.
```

Comment:
Technically they are not important for the security of the code, the code could be secure or not without this text.

```suggestion
This section argues about some aspects of the security of the code, as for example rounding issues.
References in square brackets can be found in the code as comments of the form `\\[*]`.
```